base/src/main/java/org/aya/concrete/parse/AyaProducer.java

```diff
@@ -552,7 +554,7 @@ private static SourcePos sourcePosForSubExpr(@NotNull SourceFile sourceFile, Seq
       visitTelescope(ctx.tele()),
       type(ctx.type(), sourcePosOf(ctx)),
       body,
-      abuseCtx == null ? ImmutableSeq.empty() : visitAbuse(abuseCtx)
+      bind == null ? null : visitBind(bind)
```

Comment:
It would be nice to use empty bind block instead of `null`s

Reply:
Yes!